devicon.json

```diff
@@ -4946,6 +4946,42 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "moleculer",
```

Comment:
Please move this to the correct place so it's in alphabetical order. See [this relevant comment](https://github.com/devicons/devicon/pull/1289#issuecomment-1198001547) and [this issue](https://github.com/devicons/devicon/issues/1327) for more info